src/main.rs

```diff
@@ -151,6 +151,7 @@ async fn serve_req(
             payload.extend_from_slice(&chunk);
         }
 
+        log::info!("/zulip-hook request body: {:?}", str::from_utf8(&payload));
```

Comment:
Not that it matters too much, but why log and not tracing? And also maybe this should be `debug!`.

Reply:
It's the same thing, we just renamed it for some reason. https://github.com/rust-lang/triagebot/blob/1cdce7513d8a9ee4d3f5c1573b78fd4db7194c69/src/main.rs#L13